lib/flame/controller/actions.rb

```diff
@@ -28,10 +30,16 @@ def actions
 			##     inherit_actions exclude: %i[edit update]
 			##   end
 			def inherit_actions(actions = superclass.actions, exclude: [])
-				(actions - exclude).each do |public_method|
-					um = superclass.public_instance_method(public_method)
-					define_method public_method, um
+				actions_to_inherit = actions - exclude
+
+				actions_to_inherit.each do |public_method|
+					method_body = superclass.public_instance_method(public_method)
+					define_method public_method, method_body
 				end
+
+				refined_http_methods.merge!(
+					superclass.refined_http_methods.slice(*actions_to_inherit)
+				)
```

Comment:
It looks like `ModuleWithActions.included`. Maybe we can somehow unify this code? Call method, for example.

Reply:
Extracted common code into `#copy_actions_from` and simplified "with_actions"